packages/yoga/src/Card/native/Card/Header.jsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const Header = styled.View``;
+
+Header.displayName = 'Card.Header';
+
+export default Header;
```

Comment:
Is it necessary to have this Header component in this moment? 'couse it is empty (with no style).